internal/schema/lexer.go

```diff
@@ -307,7 +307,7 @@ func lexBlockComment(l *lexer) stateFn {
 func lexStringLiteral(l *lexer) stateFn {
 	r := l.next()
 	l.ignore()
-	l.acceptRun(digits + letters)
+	l.acceptRun(digits + letters + "@/-")
```

Comment:
hm shouldn't we accept basically anything except unescaped `"` here anyways?

Reply:
Yes, we can do that.